__tests__/test-data.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("General errors", () => {
+  test("404: responds with status code 404  with appropriate message when given a unavailable route", () => {
+    request(app)
+      .get("/api/bad-route")
+      .expect(404)
+      .then((response) => {
+        const errorMessage = response._body.msg;
+        expect(errorMessage).toBe("Route Not Found");
+      });
+  });
+});
+
+describe("GET /api/topics", () => {
+  test("200: responds with status code 200 with valid request", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns an array that is not empty", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((response) => {
+        const rows = response._body.topics;
+        expect(rows.length).toBe(3);
+        expect(Array.isArray(rows)).toBe(true);
+      });
+  });
+  test("each item in array is an object", () => {
```

Comment:
As mentioned in the previous review, this test is covered implicitly in the next test so is unnecessary